example-get-started/README.md

```diff
@@ -24,10 +24,10 @@ For the basic use case (docs and Studio demo), use the command below.
 ```
 
 If change source code, to publish it on S3 (needed for the get started tutorial)
-pass `true` to the command. It's needed when you ready to publish it.
+pass `prod` to the command. It's needed when you ready to publish it.
 
 ```shell
-./generate.sh true
+./generate.sh prod
```

Comment:
Open to changing this, but there are some checks that the arg has the value `prod`